### DISCUSSION TITLE
Login and Signup Page

### DISCUSSION THREAD

**Discussion by Momin-786** [Momin-786/Stormly_WeatherWeb#16] [Q&A] [answered]
I want to create Login and signup page. tell me how to create it . THANKS IN ADVANCE

**Comment by KashifKhn** [comment, marked as answer]
in this project the Dev use the Vanilla css so you have to create new page of loing and make a UI components and style it with css

**Comment by KashifKhn** [comment]
in this project the Dev use the Vanilla css so you have to create new page of signup and make a UI components and style it with css
